janus_core/calculations/md.py

```diff
@@ -177,8 +174,7 @@ class MolecularDynamics(BaseCalculation):
 
     def __init__(
         self,
-        struct: Atoms | None = None,
-        struct_path: PathLike | None = None,
+        struct: Atoms | PathLike,
```

Comment:
as above

Reply:
Same as the others, I think this is fine